README.md

```diff
@@ -126,13 +134,14 @@ potentially, overlap the zones below.
 
 ![Drag item edit](/doc/img/edit-view-items.png)
 
-In the final step, you define the background and text color for drag
-items, as well as the drag items themselves. A drag item can contain
-either text or an image. You can define custom success and error
-feedback for each item. The feedback text is displayed in a popup
-after the learner drops the item on a zone - the success feedback is
-shown if the item is dropped on a correct zone, while the error
-feedback is shown when dropping the item on an incorrect drop zone.
+In the final step, you define the background and text color for drag items, as
+well as the drag items themselves. A drag item can contain either text or an
+image. You can define custom success and error feedback for each item. In
+standard mode, the feedback text is displayed in a popup after the learner drops
+the item on a zone - the success feedback is shown if the item is dropped on a
```

Comment:
No; the buckets unit demo does not have success/failure feedback defined for any of its items:

![screen shot 2016-08-11 at 1 46 51 pm](https://cloud.githubusercontent.com/assets/945577/17604313/1ebc630a-5fca-11e6-9d86-2af07906153f.png)

Try the "[Canada Demo](http://studio-dndv2-sandbox8.opencraft.hosting/container/block-v1:Opencraft+DNDv2+v2.1+type@vertical+block@4efd52dbdad6498ba51bb7809aa12ca1)" and drag "British Columbia" onto somewhere wrong.